python/lsst/meas/base/noiseReplacer.py

```diff
@@ -382,7 +382,7 @@ class ImageNoiseGenerator(NoiseGenerator):
 
     def __init__(self, img):
         """
-        img: an afwImage.ImageF
```

Comment:
I realize changes are coming, but was just trying to make the file internally consistent (but forgot the !).  I can revert if you think that's preferable.